R/common.R

```diff
@@ -80,6 +80,7 @@ runJaspResults <- function(name, title, dataKey, options, stateKey, functionCall
     dataset <- do.call(.readDataSetToEnd, cols)
   }
 
+  registerFonts()
```

Comment:
This might be good to perform when a new R session is started inside `initEnvironment()`

Reply:
That would ensure that it always exists, which is a good idea! But to update it, the current constructions with the `jasp_rcpp_... ` functions might still be necessary.

Reply:
> 
> 
> This might be good to perform when a new R session is started inside `initEnvironment()`

https://github.com/jasp-stats/INTERNAL-jasp/issues/1345